src/Credentials/CredentialProvider.php

```diff
@@ -267,6 +265,27 @@ public static function ecsCredentials(array $config = [])
     }
 
     /**
+     * Remote credentials provider returns a credentials provider for the
+     * default remote endpoints such as EC2 or ECS Roles.
+     *
+     * @param array $config Array of configuration data.
+     *
+     * @return InstanceProfileProvider|EcsCredentialProvider
+     * @see Aws\Credentials\InstanceProfileProvider for $config details.
+     * @see Aws\Credentials\EcsCredentialProvider for $config details.
+     */
+    public static function remoteCredentialsProvider(array $config = [])
+    {
+        $ecsCredentialsUri = getenv(EcsCredentialProvider::ENV_URI);
+
+        if (!$ecsCredentialsUri) {
```

Comment:
How about  changing this to one line like:
``` php
if(empty(getenv(EcsCredentialProvider::ENV_URI))){
    ...
```
In this way, if the `ENV` present with an empty string, it will fail quickly as well.